virl.jinja

```diff
@@ -178,7 +178,8 @@
     'l2_address2_iponly' : salt['pillar.get']('virl:l2_address2', salt['grains.get']('l2_address2', '172.16.2.254/xx' )).split('/')[0],
     'l2_address_iponly' : salt['pillar.get']('virl:l2_address', salt['grains.get']('l2_address', '172.16.1.254/xx' )).split('/')[0],
     'l3_address_iponly' : salt['pillar.get']('virl:l3_address', salt['grains.get']('l3_address', '172.16.3.254/xx' )).split('/')[0],
-    'virl_type' : salt['grains.get']('virl_type', 'stable' )
+    'virl_type' : salt['grains.get']('virl_type', 'stable' ),
+    'keystone_auth_version' : salt['pillar.get']('virl:keystone_auth_version', salt['grains.get']('keystone_auth_version', 'v2.0' ))
```

Comment:
this is already in the file and you do realize having a dupe will break shit..right